File: classes/parte-9-ingenieria-de-software-poliglota/152-rendimiento-y-perfilado-profiling/implementaciones/go/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	line, _ := bufio.NewReader(os.Stdin).ReadString('\n')
	n, _ := strconv.Atoi(strings.TrimSpace(line))
	ops, suma := 0, 0
	for i := 1; i <= n; i++ {
		suma += i
		ops++
	}
	fmt.Printf("operaciones=%d resultado=%d\n", ops, suma)
}

File: classes/parte-9-ingenieria-de-software-poliglota/152-rendimiento-y-perfilado-profiling/implementaciones/rust/main.rs
use std::io::Read;

fn main() {
    let mut s = String::new();
    std::io::stdin().read_to_string(&mut s).unwrap();
    let n: i64 = s.trim().parse().unwrap();
    let mut ops = 0i64;
    let mut suma = 0i64;
    for i in 1..=n {
        suma += i;
        ops += 1;
    }
    println!("operaciones={ops} resultado={suma}");
}
